src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -83,10 +83,11 @@ public Serializable getDefaultValue( final IdentifyingAttributes comp, final Str
 	}
 
 	public boolean isDefault( final String tag, final String attribute, final String attributeValue ) {
-		if ( attributeValue == null || attributeValue.trim().isEmpty() ) {
+		final String defaultValue = getDefaultValue( tag, attribute );
```

Comment:
>If we define a default value, than null or "" is not accepted as default

What exactly do you mean by that? In [our test](https://github.com/retest/recheck-web/blob/master/src/test/java/de/retest/web/DefaultValuesProviderTest.java#L21) we also check for `null` and `""`, which is accepted as a default. If you mean something different, could you add a test?